### DISCUSSION TITLE
Color Palette 작업 - 디자인 기반

### DISCUSSION THREAD

**Discussion by healtheloper** [Co-Studo/Co-Studo-front#39] [General]
## 예상 동작

https://coolors.co/

기본 디자인 색상은 500 에서 시작 
ex) red500

**Comment by healtheloper** [comment]
@jindonyy @hemudi @mina-gwak 

https://coolors.co/f5f5f5-007aff-ff5f57

기본 디자인 색상을 500 으로 잡고자 했으나 Shade 를 눌렀을 시 색상의 위치가 모두 다릅니다.

<img width="474" alt="스크린샷 2022-10-06 오전 1 15 42" src="https://user-images.githubusercontent.com/58503584/194110228-0384c847-c8c3-4dc0-a0b3-03b6f6e68c20.png">

<img width="489" alt="스크린샷 2022-10-06 오전 1 15 58" src="https://user-images.githubusercontent.com/58503584/194110284-77cea16f-6765-4746-9810-a237ebceefd2.png">

때문에 위에서 부터 2칸을 기준으로 100 을 잡아서 진행하려 하는데 어떠신가요?

f5f5f5 -> grey100
007aff -> blue650

**Comment by jindonyy** [comment]
500이라고 정했던게 자주 쓰는 메인 색상들이니 그 색상을 기준으로 잡은거라고 생각을 했어요.
그런데 지금처럼 자주 쓰는 grey는 100이고 blue는 650 red는 450 이런 숫자들을 기억해야하는게 개인적으로는 불편하더라구요. 숫자를 잊어버렸을 때 palette 파일을 열어서 해당 색상 코드가 숫자 몇이였는지 확인하고 다시 쓰고 이랬던 기억이 있어서..?
그래서 아마 메인 컬러들은 지금 blue처럼 거의 중간쯤에 있을거라 500을 기준으로 해서 기억하기 쉽게 하고,
위나 아래로(100, 900) 위치가 있는 색상들은 무채색 계열이지 않을까 싶어요. 하나의 팔레트 안에 흰색, 회색, 검정색 3개의 색상이 들어가야하니까..??
그래서 white, grey, black 이렇게 3등분해서 하면 어떨까 했는데 지금 적어놓은 코드를 보니 어두운 white라고 하기엔 또 어두운거 같기도 하네요.. 흠..
아니면 지금은 밝은 회색이고 메인 페이지에 활성화된 탭은 어두운 회색이니 lightGrey, darkGey로 나눠서하는건 어떤가요?
(말이 길어졌네여 @_@.. 이해 안되면 질문주세여..)

**Reply by healtheloper** [reply]
lightGrey, darkGrey 를 쓰면 숫자를 쓰는 의미가 어느정도는 사라진다고 생각해요
숫자를 기억하려면 불편하겠지만 blue650 = primary, red450 = warning 이런식으로 이름을 명시하는 palette 를 따로 두면 괜찮지 않을까요?

**Reply by jindonyy** [reply]
음.. 그러네요! 주요 색상들은 palette에 저장해놓고 palette에 없는 색상들을 쓸 때는 colors에서 확인해보는 방식인거죠?? 괜찮을거 같아요. 사용해보면서 불편함을 찾으면 또 개선해보는거도 좋을듯 하네요~!

**Comment by hemudi** [comment]
좋습니당!
위의 도니 말씀처럼 사용해보다 생기는 불편함은 차차 개선해나가도 괜찮을 것 같네요!
